BlueprintUI/Sources/Environment/Keys/LayoutDirectionKey.swift

```diff
@@ -19,7 +19,8 @@ extension Environment {
 
     private enum LayoutDirectionKey: EnvironmentKey {
         static var defaultValue: LayoutDirection {
-            LayoutDirection(UIApplication.shared.userInterfaceLayoutDirection)
+            // This will be updated in BlueprintView.makeEnvironment()
+            .leftToRight
```

Comment:
Could we use `.current` (https://developer.apple.com/documentation/uikit/uitraitcollection/3238080-current) here, or is that unsafe?

Reply:
Good call!